pkg/apis/hive/v1alpha1/clusterdeployment_types.go

```diff
@@ -238,6 +238,15 @@ const (
 	IngressCertificateNotFoundCondition ClusterDeploymentConditionType = "IngressCertificateNotFound"
 )
 
+// AllConditions is a slice containing all condition types. This can be used for dealing with
+// cluster deployment conditions dynamically.
+var AllConditions = []ClusterDeploymentConditionType{
```

Comment:
s/AllConditions/AllClusterDeploymentConditions/
(to avoid confusion with other types' conditions)

Reply:
Updated, thanks.